vm-systemd/user/pulseaudio.service.d/30_qubes.conf

```diff
@@ -1,4 +1,6 @@
 [Unit]
-#ConditionNull=false
-ConditionPathExists=/var/run/qubes-service/pulseaudio-vanilla-broken
+ConditionPathExists=/etc/pulse/qubes-default.pa
 
+[Service]
```

Comment:
The service needs also take care about waiting for audiovm to start: https://github.com/QubesOS/qubes-gui-agent-linux/commit/cabf69ffb8abacac0c12ef929637f5e223f4a1dd#diff-3c748b2e4495de699851b8c7b2144314e463fd40f14e04453c60c8fb87fa86c6R30 (and previous commit about that part). Alternatively, this can be moved into pulseaudio plugin itself. I guess PipeWire agent needs similar part too.

Reply:
For PipeWire my plan is to do what I did with Wayland, which is to treat every connection as a reconnection.  This avoids having to wait for the audio daemon to connec.

Reply:
Regardless of the final solution (the above idea sounds fine), please do the minimum required to not regress functionality here. If moving the waiting to the pulseaudio plugin itself is simple enough, go for it, otherwise add waiting as `ExecStartPre` or similar (in that case verify interaction with various timeouts - better to assume it may take at least few minutes to start).

Reply:
I don’t see any timeouts.